tests/unit/expand_conditions_spec.lua

```diff
@@ -1,8 +1,13 @@
 local helpers = require("test.functional.helpers")(after_each)
-
-local works = function(snippets, opts) end
+local ls_helpers = require("helpers")
 
 describe("expand_conditions", function()
+	before_each(function()
+		helpers.clear()
+		ls_helpers.session_setup_luasnip()
+		helpers.exec_lua("noop = function() end")
+	end)
```

Comment:
Ah, I meant `before_each` for the `extend_decorator`-tests, the `vim`-global from the last test here isn't cleared in there

Reply:
Ah now I see, maybe we should simply restore `vim` after the tests in `conditions_spec.lua` as well. But I've added `before_each` in `extend_decorator` as well (hopefully like you meant)

Reply:
Yup, that's alright :+1: